src/libraries/System.Security.Cryptography.Xml/src/System/Security/Cryptography/Xml/Utils.cs

```diff
@@ -720,10 +720,17 @@ internal static X509Certificate2Collection BuildBagOfCerts(KeyInfoX509Data keyIn
             return collection;
         }
 
+#if NET5_0_OR_GREATER
+        internal static string EncodeHexString(byte[] sArray)
+        {
+            return Convert.ToHexString(sArray);
+        }
+#else
         internal static string EncodeHexString(byte[] sArray)
         {
             return HexConverter.ToString(sArray);
         }
+#endif
```

Comment:
This doesn't seem need an if-def, it was added same time as `Convert.ToHexString(...)` added, was it failing in other builds?
```suggestion
        internal static string EncodeHexString(byte[] sArray)
        {
            return Convert.ToHexString(sArray);
        }
```

Reply:
From [nuget](https://www.nuget.org/packages/System.Security.Cryptography.Xml/#supportedframeworks-body-tab) the `System.Security.Cryptography.Xml 8.0.0` package is still targeting .net framework, while `Convert.ToHexString` is added in net5.0, so I'm not sure if it will build (or works).

Reply:
Please check the CI result https://github.com/dotnet/runtime/pull/92483/checks?check_run_id=18807551051 , removing the if-def will fail the build.

Reply:
Thanks for checking, the file doesn't included in .netframework build, probably failing on netstandard